src/components/RootApp.tsx

```diff
@@ -4,17 +4,20 @@ import { HistoryRouterProps } from 'react-router-dom-v5-compat';
 import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
 
 import { useInitApp } from '../hooks/useInitApp';
+import '../i18n';
```

Comment:
no need as it is already imported in[ useInitApp hook ](https://github.com/openshift-assisted/assisted-installer-app/blob/main/src/hooks/useInitApp.ts#L4)
```suggestion

```